### PR TITLE
fake login server

### PR DESCRIPTION
**Pull request by goofballLogic**

### CODE REVIEW

File: fake-server/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    var policy = "";
+    if(!username || !password) {
+        res.status(400).send("Bad request");
+    } else {
+        let user = fakeUsers[username];
+        if(!user) {
+            res.status(403).send("Login failed");
+            console.log("Login failed for unknown user", username);
+        } else {
+            if(user.password = password) {
```
Comment (szantilas87) [inline fake-server/index.js:21]: if (user.password === password) {

**Review by marcin-marciszewski — COMMENTED** [review]
In index.js line 21 "===" should be used instead of "="

File: fake-server/index.js (hunk 1 above)
Comment (goofballLogic) [reply fake-server/index.js:21]: ok thanks! Happy to fix any other errors you spot. @steviej08 will also review to find what he can.

Comment (goofballLogic) [reply fake-server/index.js]: fixed

**Review by marcin-marciszewski — APPROVED** [review]

**Comment by goofballLogic** [comment]
I'm duplicating this as a new Pull Request so @steviej08 has a change to review it.

**Comment by goofballLogic** [comment]
Restored as #6

**Pull request merged**